### PR TITLE
Add sentry for error reporting.

### LINKED ISSUE #7: Add error tracking mechanism
Sentry could be used for error tracking. It would definitely help in bug fixes.

### PR DESCRIPTION
**Pull request by Rishabh570**
Fixes #7 
Added sentry for error tracking.

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -257,22 +257,22 @@ parcelRequire = (function(e, r, t, n) {
         ],
         PiJs: [
             function(require, module, exports) {
-                const e = require('./fetcher'),
+                const e = require('delay'),
```
Comment (himanshuc3) [inline dist/index.js]: Is this file needed to be commited?

File: dist/index.js.map [hunk 2]
```diff
@@ -1 +1 @@
...
...etcher.getLiveMatches());\n    if (liveMatchesErr) {\n\t\tSentry.captureException(liveMatchesErr);\n        output.throwError('Could not fetch live matches, please try again!');\n\t}\n\n    // Filter out matches according to the user's query\n    let matches = helper.getMatches(liveMatches, type);\n    if (matches.length == 0) {\n        spinnerFirst.stop(); // Stop the spinner\n        output.throwNoMatchesError(type);\n    }\n\n    // Create a list of choices that user needs to select from\n    const choices = helper.makeChoicesFromMatches(matches);\n\n    spinnerFirst.stop(); // Stop the spinner\n\n    // Prompt the user to select a match from the list of choices\n    const [match, matchErr] = await handler(prompts.askForMatch(choices));\n    if (matchErr) {\n\t\tSentry.captureException(matchErr);\n\t\toutput.throwError('Could not fetch data for the match, please try again!');\n\t}\n\n    // TODO: ADD SPINNER HERE (THINK ABOUT EXPORTING THE SPINNER OBJECT)\n\n    const ID = matches[choices.indexOf(match)].match_id; // ID of the match user selected\n    matchObs(ID); // Create an observable that keeps track of the stats for the desired match and shows them to the user\n})();\n"]}
```
Comment (himanshuc3) [inline dist/index.js.map]: again, not needed I think.

File: dist/index.js (hunk 1 above)
Comment (Rishabh570) [reply dist/index.js]: Okay, so as of now parcel-bundler seems to have a problem in this bundled file (implicit declaration of `parcelRequire`), so if anyone builds and runs the utility it won't work unless he/she manually prepends let/var to the file. So I guess we push this file (I've updated this to be in a working condition) and mention this in the readme as well. 

I guess we need to do this manually until we come across some workaround. :fearful:

File: dist/index.js.map (hunk 2 above)
Comment (Rishabh570) [reply dist/index.js.map]: I've removed this file.

**Review by himanshuc3 — APPROVED** [review]
LGTM
